Musicplayer.php

```diff
@@ -29,11 +29,11 @@
 		<div class="navigation">
 			<!-- Set a div that contain logo for the music database -->
 			<div class="logo">
-				<a href = "homepage.php"><img src = "images/logo8.png" height = "100"></a>
+				<a href = "#"><img src = "images/logo8.png" height = "100"></a>
```

Comment:
26/09/2021: I decided to remove the link for the logo .because I think you want to go to the home they could just click on the home page link.